r/DESCRIPTION

```diff
@@ -97,3 +97,5 @@ Collate:
     'reexports-tidyselect.R'
     'schema.R'
     'util.R'
+Remotes:
```

Comment:
I don't think we want this in DESCRIPTION. We don't Suggest pkgdown as it is. This will increase build times significantly.

To build with this in our nightly docs builds (probably good to always be on latest pkgdown there anyway to detect issues), edit https://github.com/ursa-labs/arrow-r-nightly/blob/master/linux/install-with-pkgdown.R#L6

To edit the Official docs job that's used at release time (if you think pkgdown won't be released to CRAN by mid-April) you can change https://github.com/apache/arrow/blob/master/ci/docker/linux-apt-docs.dockerfile#L96